tests/ui/or-patterns/exhaustiveness-unreachable-pattern.rs

```diff
@@ -131,22 +165,25 @@ fn main() {
     // https://github.com/rust-lang/rust/issues/76836
     match None {
         Some(false) => {}
-        None | Some(true
-                | false) => {} //~ ERROR unreachable
+        None | Some(true | false) => {} //~ ERROR unreachable
```

Comment:
this formatting was on purpose, to make sure the span doesn't point to the `None` or the `true`